apps/web-giddh/src/app/routes-array.ts

```diff
@@ -1,6 +1,9 @@
 export const ROUTES = [
+    { path: 'download' },
```

Comment:
This file is to compare the existing routes with the last state, so ```component``` is not required